RecoTauTag/RecoTau/test/runDeepTauIDsOnMiniAOD.py

```diff
@@ -0,0 +1,46 @@
+# produce pat::Tau collection with the new DNN Tau-Ids from miniAOD 12Apr2018_94X_mc2017
+
+import FWCore.ParameterSet.Config as cms
+from FWCore.ParameterSet.VarParsing import VarParsing
+
+options = VarParsing('analysis')
```

Comment:
Is `VarParsing` intended to be used in any way in this example? If not, I would remove it - please comment.